Makefile

```diff
@@ -502,6 +502,12 @@ lint/markdown:
 	pnpm exec markdownlint-cli2 --fix "../docs/**/*.md"
 .PHONY: lint/markdown
 
+lint/md-tables:
+	cd site
+	../scripts/pnpm_install.sh
+	pnpm exec markdown-table-formatter "../docs/**/*.md"
+.PHONY: lint/markdown
```

Comment:
Its a formatter so consider moving this to `fmt/mardkown`. WDYT?

Reply:
done https://github.com/coder/coder/pull/15849/commits/8dfa01f0d31bc83e6e4f803f8a8ad159db095b52